packages/protocol/test/governance/blockchainparams.ts

```diff
@@ -0,0 +1,19 @@
+import { BlockchainParamsContract, BlockchainParamsInstance } from 'types'
+
+const BlockchainParamsTest: BlockchainParamsContract = artifacts.require('BlockchainParams')
+
+contract('BlockchainParams', () => {
+  let blockchainParamsTest: BlockchainParamsInstance
+  const version = '1.8.4'
+
+  beforeEach(async () => {
+    blockchainParamsTest = await BlockchainParamsTest.new()
+  })
+
+  describe('#setMinimumClientVersion()', () => {
```

Comment:
Add tests for `initialize`. For both functions, include a check for emitted events.